vnext/Microsoft.ReactNative.Cxx.UnitTests/ReactModuleBuilderMock.h

```diff
@@ -7,13 +7,12 @@
 #include "future/future.h"
 
 #include <functional>
+#include "..\include\CppWinRTIncludes.h"
```

Comment:
nit: Please use forward slashes for the includes.
We had issues in the past with the back slashes targeting other platforms. It maybe not an issue if we always run this code on Windows and use MSVC. It is probably just matter or style in this case.